scripts/reset_localization.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'os'
+require_relative '../RubySetupSystem/RubyCommon'
+
+def editor
+  return 'poedit' unless which('poedit').nil?
+
+  core_editor = `git config --global core.editor`
+  return core_editor unless core_editor.nil?
```

Comment:
Is it possible for the backtick syntax to return nil? I don't think I've seen that happen. I'd use instead something like
```
stuff = `some command`.strip
if stuff == ''
  do stuff...
end
```